pkg/utils/authenticationToken/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package authenticationtokenutils contains shared utility methods for the management of the
+// authentication token.
+package authenticationtokenutils
```

Comment:
Would personally shorten to authtoken (utils implied by parent folder). [Package names should be short](https://go.dev/blog/package-names)